stylesheets/_touch-friendly-links.scss

```diff
@@ -0,0 +1,10 @@
+// This mixin will expand the clickable area for a link on mobile and tablet to make it easier to click on a touch
+// device.
+
+@mixin touch-friendly-links($padding: 3px) {
+  a {
+    padding: $padding;
+    margin: -1 * $padding;
+    outline: none;
```

Comment:
Okay so it can be done - but _why_ would people do it? If they're visually impaired and want to crank contrast? Or should that have been thought about in design already?